Fix:
error: implicit declaration of function 'yylex' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'yyerror' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- clientparse.y.orig	2021-11-24 07:57:31.000000000 -0600
+++ clientparse.y	2022-01-17 03:37:39.000000000 -0600
@@ -23,6 +23,9 @@
 #define YYDEBUG 1
 #define YYERROR_VERBOSE
 
+int yylex(void);
+void yyerror(const char *);
+
 static dictServer *s;
 %}
 
--- servparse.y.orig	2021-11-24 07:57:31.000000000 -0600
+++ servparse.y	2022-01-17 03:38:48.000000000 -0600
@@ -28,6 +28,9 @@
 #define YYDEBUG 1
 #define YYERROR_VERBOSE
 
+int yylex(void);
+void yyerror(const char *);
+
 static dictDatabase *db;
 
 static int string2bool (const char *str)
